### DISCUSSION TITLE
How to host Saleor in a remote development environment?

### DISCUSSION THREAD

**Discussion by manuelaguirre** [saleor/saleor#8293] [Deployment]
Hi, in our office we are testing Sailor. I deployed it using Docker Compose in an on-premises server, but I'm encountering a problem which I could not solve. When building and deploying I connect to `<SERVER_HOST>:9000` I get served the Login Page, but when I click on connect with the credentials, the app makes a request to `localhost:8000/graphql/` which naturally is inactive in my local machine. I would like it to point to `<SERVER_HOST>:9000/graphql/`. I tried to build a production but I could only get the core running.

**Comment by xydac** [comment]
@manuelaguirre  I had the same issue when I was trying it with docker compose, I went around this by adding API_URI env value for storefront and dashboard, and used an Nginx in front to wire all services together

**Reply by manuelaguirre** [reply]
Hi @xydac  thank you, where did you add API_URI?
